generators/client/templates/angular/src/main/webapp/app/polyfills.ts.ejs

```diff
@@ -18,4 +18,5 @@
 -%>
 import 'core-js/proposals/reflect-metadata';
```

Comment:
it just came to my mind that we also don't require it any more as we would be using ngtools to build with ivy